include/mqt-core/dd/UniqueTable.hpp

```diff
@@ -195,7 +170,7 @@ class UniqueTable {
 
         while (p != nullptr) {
           std::cout << "\t\t" << std::hex << reinterpret_cast<std::uintptr_t>(p)
-                    << std::dec << " " << p->ref << std::hex;
+                    << std::dec;
```

Comment:
This can be dropped entirely, otherwise the subsequent pointers would not be printed in hex.